packages/runtime/src/builder/serialization/controls/style-v2.ts

```diff
@@ -0,0 +1,24 @@
+import { StyleV2ControlDefinition } from '../../../controls'
+import { Serialize, Deserialize } from './types'
+
+export function serializeStyleV2Control(
+  control: StyleV2ControlDefinition,
+): [Serialize<StyleV2ControlDefinition>, Transferable[]] {
+  const { getStyle, ...config } = control.config
+
+  const transferables: Transferable[] = []
+  return [
+    {
+      ...control,
+      config,
+    },
+    transferables,
+  ] as [Serialize<StyleV2ControlDefinition>, Transferable[]]
+}
+
+export function deserializeStyleV2Control(
+  control: Serialize<StyleV2ControlDefinition>,
+): Deserialize<Serialize<StyleV2ControlDefinition>> {
+  // TODO: `as unknown` shouldn't be required here 
```

Comment:
I don't understand how the function serialization is working ATM. 

I didn't think we could serialize functions, but there are types (`SerializedFunction` etc) for serialized functions. 

Just admitting this because I don't really know where to start in fixing this `as unknown`.

Reply:
We serialize functions by serializing arguments and return values and sending them via `postMessage`. This means that serialized functions become `async`. An example of this is the `getItemLabel` function in the `List` control which we use to render a label in the builder panel